backend/Dockerfile

```diff
@@ -1,16 +1,19 @@
-FROM node:18-alpine
-
-# Set container working directory
-WORKDIR /app
-
-RUN apk update
-RUN apk upgrade
-RUN apk add bash
-
-RUN npm install -g npm@8.8.0
-
-EXPOSE 3000
-
-COPY ./docker-entrypoint.sh /tmp/
-
-CMD ["bash", "/tmp/docker-entrypoint.sh"]
+FROM node:18-alpine
+
+# Set container working directory
+WORKDIR /app
+
+RUN apk update
+RUN apk upgrade
+RUN apk add bash
+
+RUN npm install -g npm@8.8.0
+
+EXPOSE 3000
+
+COPY ./docker-entrypoint.sh /tmp/
+
+ARG WATCH=1
+ENV WATCH=$WATCH
```

Comment:
Won't this set `WATCH` to always be `1`, even if it is unset in the `.env` file?

Reply:
No, WATCH, as a build argument would have value `1`, but it only provides a default value to WATCH the environment variable.